.kokoro/nightly/downstream-protobuf-binary-compatibility.sh

```diff
@@ -65,7 +65,7 @@ function build_program_arguments() {
 # cloud-opensource-java contains the Linkage Checker tool
 git clone https://github.com/GoogleCloudPlatform/cloud-opensource-java.git
 pushd cloud-opensource-java
-mvn -B -ntp clean compile -T 1C
+mvn -B -ntp clean compile -T 1C -Djvm="${JAVA8_HOME}/bin/java"
```

Comment:
Turns out the `-Djvm` flag is specific to Surefire not Maven. This still compiles with JDK 17.